### ISSUE TITLE
[data request] Flying Chairs

### ISSUE THREAD

**Issue by Ecalpal** [tensorflow/datasets#97]
* Name of dataset: Flying Chairs
* URL of dataset: https://lmb.informatik.uni-freiburg.de/resources/datasets/FlyingChairs.en.html
* License of dataset: 
Not noted，
"Terms of Use
The dataset is provided for research purposes only and without any warranty. Any commercial use is prohibited. If you use the dataset in your research, you should cite the aforementioned paper."
* Short description of dataset and use case(s): It is a synthetic dataset with optical flow ground truth.We usually use it to train the optical flow network.

Folks who would also like to see this dataset in `tensorflow/datasets`, please +1/thumbs-up so the developers can know which requests to prioritize.

**Comment by PulkitMishra** [comment]
@rsepassi @Ecalpal , I'd like to work on this issue. Have gone through gitter discusions and [this](https://twitter.com/DynamicWebPaige/status/1099772611070816256) thread.

**Comment by adikolsur** [comment]
@Ecalpal I would like to contribute to this dataset. Kindly assign this project to me, please.

**Comment by rsepassi** [comment]
Thanks for your interest! @PulkitMishra, let me know when you've accepted the invite, and I'll assign this to you. @adikolsur, let's see if we can find you another dataset to work on. Here are the current [unassigned dataset requests](https://github.com/tensorflow/datasets/issues?q=is%3Aissue+is%3Aopen+label%3A%22dataset+request%22+no%3Aassignee) (though for some, somebody may have already requested it be assigned to them, so check the comments).

**Comment by PulkitMishra** [comment]
@rsepassi I have accepted the invite and am good to go. Any heads up?

**Comment by rsepassi** [comment]
Great, looking forward to the PR! Can start with [Flying Chairs](https://lmb.informatik.uni-freiburg.de/resources/datasets/FlyingChairs.en.html#flyingchairs).

**Comment by Ecalpal** [comment]
Thank you all!  @rsepassi @PulkitMishra @adikolsur 
Because there are not many people in the field of optical flow, I really appreciate your help.
It took me a lot of time to write optical flow dataset when I training an optical flow such as pwc-net in tensorflow.
so, when I see TensorFlow Datasets, I admire the fact that people in other fields can directly use the dataset provided by tensorflow.
In fact, I just asked for a basic dataset, and other datasets such as Flyingtings3D, MPI Sintel are also very important. Should I create other issues?

**Comment by rsepassi** [comment]
Yes, please do create additional dataset requests for the datasets you'd like to see @Ecalpal (and we'd love for you to contribute since it sounds like those may be tricky to get right!).

**Comment by Ecalpal** [comment]
I wish I could join in, but I'm too busy with my studies. Once I'm free, I'll join in. 
by the way, tensorflow is a great work, Looking forward to tensorflow 2！

**Comment by PulkitMishra** [comment]
@rsepassi I am guessing that will this require the creation of a new feature in FeatureConnector?
Also, apologies for the inactivity (was busy with a hackathon)

@Ecalpal

**Comment by adriaangraas** [comment]
I've written this [FlyingChairs dataset](https://github.com/adriaangraas/datasets/tree/flying-chairs-dataset).

The [original dataset split](https://lmb.informatik.uni-freiburg.de/resources/datasets/FlyingChairs/FlyingChairs_train_val.txt) provides only training and validation examples, there are no testing samples. Should I go ahead and provide an own split or is this fine?

What can I do to proceed?

**Comment by Ecalpal** [comment]
> I've written this [FlyingChairs dataset](https://github.com/adriaangraas/datasets/tree/flying-chairs-dataset).
> 
> The [original dataset split](https://lmb.informatik.uni-freiburg.de/resources/datasets/FlyingChairs/FlyingChairs_train_val.txt) provides only training and validation examples, there are no testing samples. Should I go ahead and provide an own split or is this fine?
> 
> What can I do to proceed?

Sorry, I haven't used tensorflow for a long time.
This dataset does not need test split.

**Comment by adriaangraas** [comment]
Hi @Ecalpal, can you reopen the issue? It is still valid for other people.
@rsepassi if you can assign me on to the issue I'll PR the dataset.
